tracer/src/Datadog.Trace/Propagators/SpanContextPropagator.cs

```diff
@@ -19,6 +19,7 @@ internal class SpanContextPropagator
     {
         internal const string HttpRequestHeadersTagPrefix = "http.request.headers";
         internal const string HttpResponseHeadersTagPrefix = "http.response.headers";
+        internal const string ZeroLastParentId = "0000000000000000";
```

Comment:
Wasn't this defined already in a previous PR?

Reply:
We did in a different class as private, I wasn't sure if I should extend it or leave the change as I added above: https://github.com/DataDog/dd-trace-dotnet/blame/master/tracer/src/Datadog.Trace/Propagators/W3CTraceContextPropagator.cs#L61